src/xcode/ENA/ENA/Resources/Localization/en.lproj/usage.html

```diff
@@ -1,448 +1,454 @@
 <!DOCTYPE html>
 <html lang="de">

```

Comment:
Why is it german for english case?

Reply:
I guess the whole file is not yet localized :-D

Reply:
Yep, leaving all the localization out of this commit!